docs/snippets/vue/page-story-slots.3.ts-4-9.mdx

```diff
@@ -5,19 +5,11 @@ import type { Meta, StoryObj } from '@storybook/vue3';
 
 import Page from './Page.vue';
 
+// TODO: How do you extract a Vue3 component's props type?
+// type PagePropsAndCustomArgs = Page & { footer?: string };
```

Comment:
We use 
https://github.com/vuejs/language-tools/tree/master/packages/vue-component-type-helpers

Not sure if we should add that to the docs though.. Hmm 🤔